DB/SharedConstants.lua

```diff
@@ -264,6 +264,59 @@ C.FACTION_IDS = {
 	ISKAARA_TUSKARR = 2511,
 }
 
+C.LFG_DUNGEON_IDS = {
+	TORGHAST_LAYER = {
```

Comment:
I was actually thinking there would be one entry per layer, and then you could use a simple lookup. Your solution should also work, although technically `tContains` may be slower as it iterates the whole table.

Up to you, I guess, since it's unlikely to make a difference in this case: The branch will only ever be executed inside Torghast, although we could theoretically try to benchmark to see if it's measurable at all. But ehh... whatever.

I would rename this to `TORGHAST_LAYERS`, though, since you're not mapping each layer but all of them.

Reply:
Update: I just noticed that holiday events are also using these IDs, which are currently hardcoded. For example, the Hallow's End or Love is in the Air events. These could be moved to the shared constants as well, which means we would probably have to go with the "one mapping per dungeon ID" line here?

Reply:
well, theres already some freezing lag in torghast so a little more cant hurt